# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/transforms/reftest.list
# Commit: 431c5142ff3a
# Full Hash: 431c5142ff3ad102328d90cb29470b20ccd1271e
# Author: Glenn Watson <github@intuitionlibrary.com>
# Date: 2019-06-13 21:50:57
# Regressor Bug: 1558106
# File Overlap Count: 1
# Description:
#   Bug 1558106 - Support picture caching for multiple scroll roots. r=kvark
#   
#   This patch implements the majority of the planned picture caching
#   improvements. It supports most of the functionality required to
#   (as a follow up) support OS compositor integration. It also improves
# ==============================================================================

diff -r 855d557b82a2 -r 431c5142ff3a gfx/wr/wrench/reftests/transforms/reftest.list
--- a/gfx/wr/wrench/reftests/transforms/reftest.list	Wed Jun 12 16:18:22 2019 +0000
+++ b/gfx/wr/wrench/reftests/transforms/reftest.list	Thu Jun 13 04:43:56 2019 +0000
@@ -1,7 +1,7 @@
 platform(linux,mac) == local-clip.yaml local-clip.png
 platform(linux,mac) == rotated-clip.yaml rotated-clip.png
 platform(linux,mac) == rotated-clip-large.yaml rotated-clip-large.png
-skip_on(android,emulator) == image-rotated-clip.yaml image-rotated-clip.png  # Fails on Android emulator
+platform(linux,mac) skip_on(android,emulator) == image-rotated-clip.yaml image-rotated-clip.png  # Fails on Android emulator
 # Something leaks the state: the test passes if only run `reftest reftests/transform`
 # but fails when all the tests are run
 platform(linux,mac) fuzzy(1,4) == rotated-image.yaml rotated-image.png
@@ -13,8 +13,8 @@
 platform(linux,mac) == coord-system.yaml coord-system.png
 platform(linux,mac) fuzzy(1,5) zoom(4) == border-zoom.yaml border-zoom.png
 platform(linux) fuzzy(1,520) == perspective-origin.yaml perspective-origin.png
-platform(linux,mac) color_targets(2) alpha_targets(0) fuzzy(1,180) == screen-space-blit.yaml screen-space-blit.png
-platform(linux,mac) fuzzy(1,331) color_targets(1) alpha_targets(0) == screen-space-blit-trivial.yaml screen-space-blit-trivial.png
+platform(linux,mac) color_targets(3) alpha_targets(0) fuzzy(1,180) == screen-space-blit.yaml screen-space-blit.png
+platform(linux,mac) fuzzy(1,331) color_targets(2) alpha_targets(0) == screen-space-blit-trivial.yaml screen-space-blit-trivial.png
 platform(linux) fuzzy(11,4592) == screen-space-blur.yaml screen-space-blur.png
 platform(linux,mac) == nested-rotate-x.yaml nested-rotate-x.png
 platform(linux,mac) != nested-rotate-x.yaml nested-rotate-x-flat.yaml